src/GraphClient.ts

```diff
@@ -46,8 +54,9 @@ export default class GraphClient {
       query: gqlSubscriptionQry,
       variables: {},
     }).subscribe({
-      next(response) {
-        handler.handle(response);
+      async next(response) {
+        const transactions = await fetcher.fetch(response.data);
+        await handler.handle(transactions);
```

Comment:
Should we add a place holder here to keep track of the place from where we would need to update last processed timestamp in uts table ?

Reply:
That place holder will be in individual entity handlers 🤝 .

Reply:
It might lead to all handlers implementing / calling same piece of code. Can we reconsider having it here ? Similar to us adding the fetch logic here. Thoughts ?

Reply:
Added comment as placeholder. It needs to be integrated with #91 after it's done